MapleServer2/Types/SkillTab.cs

```diff
@@ -1,74 +1,54 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
-using System.Runtime.InteropServices;
-using MaplePacketLib2.Tools;
 
-namespace Maple2.Data.Types {
-    public class SkillTab : IByteSerializable {
+namespace MapleServer2.Types {
```

Comment:
Is there any reason we changed the namespace?

Reply:
Yes, that path no longer exists, the file is under MapleServer2/Types, it was previously copy pasted from a different directory without changing namespace before I touched the file.